plugins/RocketsPlugin/RocketsPlugin.cpp

```diff
@@ -301,6 +307,33 @@ class RocketsPlugin::Impl
                                                std::placeholders::_1));
     }
 
+    void _delayedNotify(const std::function<void()>& notify)
+    {
+        std::lock_guard<std::mutex> lock(_delayedNotifiesMutex);
+
+#ifdef BRAYNS_USE_LIBUV
+        if (uvw::Loop::getDefault()->alive())
+#else
+        if (false)
+#endif
+        {
+#ifdef BRAYNS_USE_LIBUV
+            // dispatch delayed notify from throttle thread to main thread
```

Comment:
I think it is more readable to make this one ifdef/else block.

Reply:
yeah, agreed